docs/designs/crd.md

```diff
@@ -492,6 +501,34 @@ Every Keystore Value defines a secret to pull secrets from.
             <th>default</th>
         </tr>
     </thead>
+    <tbody><tr>
```

Comment:
This is off by a few lines. The table for the monitoring fields is currently under the keystore section, needs to be moved to the monitoring section.